contracts/src/deployers/steth/StETHHyperdriveCoreDeployer.sol

```diff
@@ -31,14 +31,13 @@ contract StETHHyperdriveCoreDeployer is IHyperdriveCoreDeployer {
         address target4,
         bytes32 _salt
     ) external returns (address) {
-        return (
-            address(
-                // NOTE: We hash the sender with the salt to prevent the
-                // front-running of deployments.
-                new StETHHyperdrive{
-                    salt: keccak256(abi.encode(msg.sender, _salt))
-                }(_config, target0, target1, target2, target3, target4)
-            )
+        address deployAddress = address(
```

Comment:
Nit: I think `hyperdrive `would be a more intuitive name.

```suggestion
        address hyperdrive = address(
```

Reply:
this just needs to be reverted